### PR TITLE
Add template outputs in aks user guide.

### PR DESCRIPTION
**Pull request by galiacheng**
Signed-off-by: galiacheng <haixia.cheng@microsoft.com>

 Changes to be committed:
	modified:   documentation/3.3/content/userguide/aks/_index.md
	modified:   documentation/3.4/content/userguide/aks/_index.md
	modified:   documentation/staging/content/managing-domains/aks/_index.md

### CODE REVIEW

**Review by edburns — APPROVED** [review]

File: documentation/3.3/content/userguide/aks/_index.md [hunk 1]
```diff
@@ -296,6 +297,31 @@ If you want to use this template to automate the deployment, download it by sele
 
 Click **Create** to create this offer. This process may take 30 to 60 minutes.
 
+#### Template outputs
+
+After clicking **Create** to create this offer, you will go to the **Deployment is in progress** page. When the deploymet is completed, the page shows **Your deployment is complete**. In the left panel, select **Outputs**. These are the outputs from the deployment.  The following table is a reference guide to the deployment outputs.
```
Comment (rosemarymarano) [inline documentation/3.3/content/userguide/aks/_index.md:302]: deploymet  -> deployment

File: documentation/3.3/content/userguide/aks/_index.md [hunk 2]
```diff
@@ -296,6 +297,31 @@ If you want to use this template to automate the deployment, download it by sele
...
+
+| Field | Description |
+|-------|-------------|
+| `aksClusterName` | Name of your AKS cluster that is running the WLS cluster. {{< line_break >}}Sample value: `wlsonaksiyiql2i2o2u2i`. |
+| `adminConsoleInternalUrl` | The fully qualified, private link to the Administration Console portal. You can only access it inside AKS cluster. {{< line_break >}}Sample value: `http://sample-domain1-admin-server.sample-domain1-ns.svc.cluster.local:7001/console`. |
```
Comment (rosemarymarano) [inline documentation/3.3/content/userguide/aks/_index.md:307]: can only access it -> can access it only 
inside AKS cluster -> inside an AKS cluster

File: documentation/3.3/content/userguide/aks/_index.md [hunk 3]
```diff
@@ -296,6 +297,31 @@ If you want to use this template to automate the deployment, download it by sele
...
+| `adminConsoleExternalUrl` | This output is not always present:{{< line_break >}}You must configure [Networking](#networking) to enable Azure Load Balancer service or Azure Application Gateway Ingress Controller for the Administration Console.{{< line_break >}} {{< line_break >}}This is a fully qualified, public link to the Administration Console portal. You can access it from the public Internet. {{< line_break >}}Sample value: `http://wlsgw202208-wlsd-aks-2793762585-337-domain1.eastus.cloudapp.azure.com/console`. |
```
Comment (rosemarymarano) [inline documentation/3.3/content/userguide/aks/_index.md:308]: to enable Azure Load Balancer service -> to enable the Azure Load Balancer service

File: documentation/3.3/content/userguide/aks/_index.md [hunk 4]
```diff
@@ -296,6 +297,31 @@ If you want to use this template to automate the deployment, download it by sele
...
+| `adminConsoleExternalSecuredUrl` | This output is not always present:{{< line_break >}}1. You must configure [Networking](#networking) to enable Azure Load Balancer service or Azure Application Gateway Ingress Controller for the Administration Console.{{< line_break >}}2. You must configure a custom DNS name by filling out [DNS Configuration](#dns-configuration).{{< line_break >}}3. The TLS/SSL certificate used is configured by filling out [TLS/SSL configuration](#tlsssl-configuration).{{< line_break >}}{{< line_break >}}This is a fully qualified, secure, public link to the Administration Console portal. You can access it from the public Internet. {{< line_break >}}Sample value: `https://contoso.com/console`. |
```
Comment (rosemarymarano) [inline documentation/3.3/content/userguide/aks/_index.md:309]: to enable Azure Load Balancer service -> to enable the Azure Load Balancer service (please make this correction wherever this text appears)

File: documentation/3.3/content/userguide/aks/_index.md [hunk 5]
```diff
@@ -296,6 +297,31 @@ If you want to use this template to automate the deployment, download it by sele
...
+| `clusterT3ExternalEndpoint` | This output is not always present:{{< line_break >}}1. You must [create/update the WLS cluster with advanced configuration](https://oracle.github.io/weblogic-azure/aks/).{{< line_break >}}2. You must enable custom T3 channel by setting `enableClusterT3Tunneling=true`.{{< line_break >}}3. You must configure [Networking](#networking) to enable Azure Load Balancer service for the WLS cluster.{{< line_break >}}{{< line_break >}}This is a fully qualified, public link to custom T3 channel of the WLS cluster.{{< line_break >}}Sample value:`http://20.4.56.3:8005/` |
+| `shellCmdtoConnectAks` | AZ CLI command to connect the AKS cluster. This enables you to use `kubectl` to interact with the cluster. {{< line_break >}}Sample value: {{< line_break >}}`az account set --subscription xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx; az aks get-credentials --resource-group contoso-rg --name contosoakscluster`|
```
Comment (rosemarymarano) [inline documentation/3.3/content/userguide/aks/_index.md:319]: to connect the AKS cluster -> to connect to the AKS cluster

**Review by rosemarymarano — COMMENTED** [review]
Few minor edits that should be repeated wherever the same text occurs. 

After your docs are merged, I can do a much more thorough review by being able to view them in the documentation UI. Just let me know if this is something you would like me to do.

**Comment by galiacheng** [comment]
Hello @rosemarymarano thanks for your helpful comments, I have applied all of them in https://github.com/oracle/weblogic-kubernetes-operator/pull/3301/commits/22a7de1315e489cc2a2953683a39a11068b73b84.

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**